README.rst

```diff
@@ -31,6 +31,31 @@ To install from source on github_::
 
 .. _github: https://github.com/chapel-lang/sphinxcontrib-chapeldomain
 
+Release
+-------
+
+#. Test and commit changes
+#. Merge your changes
```

Comment:
This should not be a part of the release process

Reply:
sure, that has been the status quo. But its not necessarily true and it seems misleading to start a section about "release process" with "make the change"

Reply:
Do you see it changing in the future?

I'd be willing to add some wording about that expectation, maybe something about "changes or sets of changes in a short period of time", but I think this documentation should reflect the most reasonable expected use case and my expectation is that releases would be motivated by single to grouped changes rather than an explicit cadence for the foreseeable future.